### PR TITLE
update test example title 

### PR DESCRIPTION
**Pull request by sudivate**
update test example title

### CODE REVIEW

File: examples/azure/terraform-azure-keyvault-example/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Terraform Azure Availability Set Example
+# Terraform Azure Keyvault  Example
```
Comment (brikis98) [inline examples/azure/terraform-azure-keyvault-example/README.md:1]: NIT: 

```suggestion
# Terraform Azure Keyvault Example
```

**Review by brikis98 — COMMENTED** [review]
Thanks! Looks like an extra space snuck in.

File: examples/azure/terraform-azure-keyvault-example/README.md (hunk 1 above)
Comment (sudivate) [reply examples/azure/terraform-azure-keyvault-example/README.md]: fixed

**Review by brikis98 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**